2021/46xxx/CVE-2021-46380.json

```diff
@@ -1,31 +1,8 @@
 {
```

Comment:
('Expecting property name enclosed in double quotes: line 18 column 1 (char 476)',)